lib/helpers/ap-service.ts

```diff
@@ -8,12 +9,25 @@ export const getApServiceSettings = (serviceType: TServiceType) => {
   return serviceSettings[serviceType as TServiceType] ?? null
 }
 
-export const getApServiceUrl = (serviceType: TServiceType) => {
+export const getApServiceUrl = async (serviceType: TServiceType) => {
   const serviceSettings = getApServiceSettings(serviceType)
   if (!serviceSettings) {
     throw new Error(`No url found for the ${serviceType} service`)
   }
 
+  if (serviceType === "fbi") {
+    const dplCmsConfig = await getDplCmsPublicConfig()
```

Comment:
Since we have defaults (null) for the other settings, if we do not get them from the cms we should also add searchProfiles to [that](https://github.com/danskernesdigitalebibliotek/dpl-go/blob/6c984e6ebbc6d8c6f0278400175196af5e38515e/lib/config/dpl-cms/dplCmsConfig.ts#L83-L96)